src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,61 @@
+package duke;
+
+import duke.commands.Command;
+import duke.exceptions.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/**
+ * The Duke class is the main class which starts the chatbot
+ */
+public class Duke {
+
+    private TaskList taskList;
+    private final Ui ui;
+    private final Storage storage;
+    private final String FILEPATH = "./data/duke.txt";
+    private final String FOLDERPATH = "./data";
+
+    /**
+     * Public constructor which initialises the ui, storage and tasklist before the chatbot runs
```

Comment:
There are some violations of the [coding standards](https://se-education.org/guides/conventions/java/intermediate.html#statements) with regards to your Javadoc comments, specifically
> In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ...

Maybe you can consider something like
```suggestion
     * Constructs a new Duke chatbot, initialising the UI, storage, and task list before running.
```

Reply:
Hi! Thank you for your comments and suggestions. Will look into them and make the necessary changes!